ci/requirements-3.6-dev.yml

```diff
@@ -53,5 +52,6 @@ dependencies:
   - xorg-libxpm
   - xorg-libxrender
   - pip:
-      # see .pre-commit-config.yaml, we pin to specific versions of black and isort
+      # see .pre-commit-config.yaml, isort pinned
```

Comment:
really we need to pin like this?

Reply:
for now I think we need it because the conflicts.
maybe also we can replace the black checking inside CI for a pre-commit check